app/services/idv/profile_maker.rb

```diff
@@ -14,12 +14,18 @@ def save_profile
         user: user,
       )
       profile.encrypt_pii(pii_attributes, user_password)
+      profile.proofing_components = current_proofing_components_to_json
```

Comment:
We should probs add something to the confirmation step spec to make sure this gets written the way we expect.